engine/node.py

```diff
@@ -11,17 +11,21 @@ class Node:
         status: current status of the node [int]
             [0 = not traversed, 1 = traversed, 2 = obstacle]
 
+        is_active: whether the node is currently in use
+
         is_border: type of node on the grid
             [0 = not on the border of the grid, 1 = on the border of the grid]
+
     """
 
-    def __init__(self, lat, long, x, y, is_border='0', status=0):
+    def __init__(self, lat, long, x, y, is_border=0, active=False, status=0, waypoint = False):
```

Comment:
What is the purpose of the new parameter waypoint?

Reply:
@paulhsuh